configure.ac

```diff
@@ -4647,6 +4647,14 @@ AC_MSG_CHECKING(LDVERSION)
 LDVERSION='$(VERSION)$(ABIFLAGS)'
 AC_MSG_RESULT($LDVERSION)
 
+# On Android the shared libraries must be linked with libpython.
+AC_SUBST(LIBPYTHON)
+if test -z "$ANDROID_API_LEVEL"; then
```

Comment:
Out of curiosity, being that I'm unfamiliar with build issues on Android why the specific test for `$ANDROID_API_LEVEL` and not something more generic like:

```
case $host in
    *-linux-android*) ....
```
?

Reason being, I need to add additional cases where libpython should be linked explicitly (e.g. Cygwin, MinGW, etc.) and it would be easier to build off the host_os for this than something very Android-specific.

Reply:
Using a test based on ``$ANDROID_API_LEVEL`` or on ``$host`` as you suggest is equivalent.